pkg/issuer/acme/http/service.go

```diff
@@ -77,9 +77,12 @@ func buildService(crt *v1alpha1.Certificate, ch v1alpha1.ACMEOrderChallenge) *co
 	podLabels := podLabels(ch)
 	return &corev1.Service{
 		ObjectMeta: metav1.ObjectMeta{
-			GenerateName:    "cm-acme-http-solver-",
-			Namespace:       crt.Namespace,
-			Labels:          podLabels,
+			GenerateName: "cm-acme-http-solver-",
+			Namespace:    crt.Namespace,
+			Labels:       podLabels,
+			Annotations: map[string]string{
+				"auth.istio.io/8089": "NONE",
```

Comment:
in case this changes, can this be read from the cert CR ?

Reply:
Perhaps in future - I'm not sure how we'd best model this in the Certificate custom resource though.

This is a pretty easy fix/quick win for Istio users, so I'm inclined to accept this PR and investigate ways we can support custom annotations in future.